app/controllers/decidim/file_authorization_handler/admin/censuses_controller.rb

```diff
@@ -26,6 +29,12 @@ def destroy
           CensusDatum.clear(current_organization)
           redirect_to censuses_path, notice: t(".success")
         end
+
+        def update
```

Comment:
In my opinion, this should not be an update. Census#update should be used to update the census, but not perform another operation that is unauthorize users.

The best approach would be create another controller: Authorizations#destroy. However, if this seems too boilerplate to you, you can also use a custom action like delete_authorizations.

Reply:
I totally agree with your opinion, so I created a custom action inside the controller.